### PR TITLE
#145115129/Local development setup

### PR DESCRIPTION
**Pull request by efalayi**
#### What does this PR do?
The PR creates a basic setup for development on local machine.

#### Description of Task to be completed?
1. Install dotenv npm module in dependencies and require it in server.js
2. Save the database url for local development  in a .env file
3. Update .gitignore file to ignore the .env created

#### How should this be manually tested?
* Clone the repo to local machine and switch to branch 'chore/145115129/development-setup
* Add the .env file
* Run 'npm install'
* Run 'npm start'

#### Any background context you want to provide?
* 'npm install' installs all required packages

#### What are the relevant pivotal tracker stories?
Story: #145115129

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,14 +45,15 @@
     "console-stamp": "~0.1.1"
   },
   "devDependencies": {
-    "supertest": "~0.8.0",
-    "should": "~1.3.0",
-    "grunt-contrib-watch": "~0.5.3",
-    "grunt-contrib-jshint": "~0.6.4",
-    "grunt-nodemon": "~0.1.1",
+    "dotenv": "^4.0.0",
```
Comment (andela-moe) [inline package.json]: I believe this should be part of your applications dependencies. Please fix this.

**Review by andela-moe — COMMENTED** [review]
Address comments

**Comment by efalayi** [comment]
dotenv module has been installed in dependencies

**Review by andela-moe — COMMENTED** [review]
Cool

**Comment by andela-moe** [comment]
Cool

**Pull request merged**